packages/clipboard/readme.md

```diff
@@ -1,3 +1,54 @@
-# Clipboard
+# @creately/clipboard
 
-Clipboard Service which uses system clipboard as primary and local storage as fallback.
+> Clipboard service which uses system clipboard as primary and local storage as fallback.
+
+Support:  All browsers and Embedded( iframe ).
+
+## Install
+
+```
+$ npm install @creately/clipboard
+```
+
+## Usage
+
+```js
+import { Clipboard } from '@creately/clipboard';
+
+new Clipboard().copy('your data');
+
+new Clipboard().paste();
+//=> 'your data'
+```
+
+If required you can use the provider to use the same instance.
```

Comment:
People can use it with any framework, not just Angular. Shall we add a `Usage with Angular` heading for this part?